reconcile/jenkins_job_builder.py

```diff
@@ -182,10 +191,22 @@ def validate_repos_and_admins(jjb, additional_repo_urls):
 
 
 @defer
-def run(dry_run, io_dir='throughput/', defer=None):
-    jjb, additional_repo_urls = init_jjb()
+def run(dry_run, print_only=False, jenkins_config=None, jenkins_job=None,
+        jenkins_instance=None, io_dir='throughput/', defer=None):
+    if jenkins_config or jenkins_job:
+        print_only = True
```

Comment:
i'm confused if `print-only` is passed in or calculated by the integration. it can get very confusing, there are so many new arguments in this PR...